src/bd/primitives/Primitive.java

```diff
@@ -0,0 +1,84 @@
+package bd.primitives;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Repeatable;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import com.oracle.truffle.api.dsl.NodeFactory;
+
+
+/**
+ * Annotation to be applied on node classes that are meant to be used as primitive operations.
+ * Thus, the node implements a built-in function of the interpreter, and might need to be
+ * represented as a language-level method/function.
```

Comment:
Ok, but actually the node having the annotation is not a language-level method. It is usually a kind of expression node. I still see the comment confusing sorry and I a still do not have a better way to communicate the idea, sorry.